src/org/labkey/test/tests/ListArchiveExportTest.java

```diff
@@ -0,0 +1,119 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.Locators;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.categories.Hosting;
+import org.labkey.test.components.list.ManageListsGrid;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ListHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@Category({Daily.class, Hosting.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class ListArchiveExportTest extends BaseWebDriverTest
+{
+    private final static String LIST_SUBFOLDER = "List subfolder";
+    private final static String LIST_FOLDER_A = "LIST_FOLDER_A";
+    private final static String LIST_A = "List Export A";
+    private final static String LIST_B = "List Export B";
+    private final static String SHARED_LIST = "Shared list C";
```

Comment:
It looks like this variable is not being used

Reply:
good catch